oshc/main/templates/contests.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block content %}
+<section class="main-section">
+	<div class="container">
+		<div class="Button" align="center">
+            <a href="{% url 'contest_new' %}" class="btn btn-default">Submit a New Contest</a>
+		</div>
+            <h3>List of Upcoming Contests:</h3>
+
+			{% for contest in contest_list %}
+                <div class="panel panel-default">
+                    <div class="panel-heading"><a href="{{contest.link}}">{{contest.name}}</a></div>
+                    <div class="panel-body">
+                        <strong>Start Date:</strong> {{contest.start_date}}<br>
```

Comment:
don't use `<br>` instead make css class and use that.

Reply:
Hi @vaibhavsingh97,
you said, "also make sure no content should come below icon (Refer screenshot)"
below which icon??